api/internal/builtins/SortOrderTransformer.go

```diff
@@ -118,11 +109,13 @@ type legacyIDSorter struct {
 	// resids only stores the metadata of the object. This is an optimization as
 	// it's expensive to compute these again and again during ordering.
 	resids     []resid.ResId
+	resources []*resource.Resource
```

Comment:
nit: Could you write what is the difference between this variable and the old variable `resids`.

Reply:
Done, I used the comment suggested by @yanniszark